notebooks/official/tensorboard/tensorboard_vertex_pipeline.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>your training code generates. -&gt; that your training code generates.</p><p> </p><p>that is, -&gt; and</p><p> </p><p>Delete "the" in the list items.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>